tests/layer1/test_port_error.py

```diff
@@ -104,7 +111,7 @@ def test_mac_remote_fault_increment(self, select_random_interfaces):
             dut.shell("sudo sfputil debug tx-output {} disable".format(interface))
             time.sleep(5)
             pytest_assert(self.get_interface_status(dut, interface) == "down",
-                          "Interface {} did not go down after disabling rx-output using sfputil".format(interface))
```

Comment:
we should use a variable in the cmd and in the print, this way we will avoid duplication and these kind of errors